content-src/components/Card/Card.jsx

```diff
@@ -200,6 +200,7 @@ export class _Card extends React.PureComponent {
   render() {
     const {index, className, link, dispatch, contextMenuOptions, eventSource, shouldSendImpressionStats} = this.props;
     const {props} = this;
+    const title = link.label || link.hostname;
```

Comment:
@AllegroFox just noticed, it seems  we don't have link.label in Card, it should be link.title || link.hostname;